oneflow/api/python/framework/tensor.cpp

```diff
@@ -233,6 +233,15 @@ void ExportTensor(py::module& m, const char* name) {
       .def_property_readonly("dtype", &GetTensorDType<T>)
       .def_property_readonly("is_cuda", &T::is_cuda)
       .def_property_readonly("grad", [](const T& t) { return t.api_acc_grad().GetPtrOrThrow(); })
+      // setter of grad
+      .def("set_grad",
+           [](T& t, const std::shared_ptr<Tensor>& grad) {
```

Comment:
```suggestion
           [](T& t, const std::shared_ptr<T>& grad) {
```

这里也用模板，避免 Local/Consistent Tensor互相赋值

Reply:
好的，请问需不需要去掉`self._undetermined_tensor`下的grad赋值呢？一开始创建tensor的时候无法得知它们的类型，所以是`_undetermined_tensor`，但一开始创建tensor的时候应该不需要grad的重新赋值？